backend/items/api.py

```
@@ -955,6 +1217,134 @@ def agent_add_item_batch(request, data: AgentBatchPromptsSchema):
+            },
+        )
+        
+        clerk_user_id = payload.get("sub")
+        if not clerk_user_id:
+            raise HttpError(401, "Invalid token")
+        
+        # Get user from database using sync_to_async
+        User = get_user_model()
+        
+        @sync_to_async
+        def get_or_create_user():
```

Ruff (0.15.2)

[warning] Missing return type annotation for private function `get_or_create_user`

(ANN202)